en/orm/saving-data.rst

```diff
@@ -759,6 +759,8 @@ table class. Behaviors and other event subscribers can use the
 ``Model.buildRules`` event to augment the rules checker for a given Table
 class::
 
+    use Cake\ORM\RulesChecker
```

Comment:
Of course antograssiot but how to do it?  Do you forget this one and I create a new pull?